blueocean-core-js/src/js/components/ReplayButton.jsx

```diff
@@ -73,13 +73,12 @@ export class ReplayButton extends Component {
         const isReplayable = this.props.latestRun.replayable;
         const hasPermission = permit(this.props.runnable).start();
 
-        const replayLabel = translate('run.rerun');
+        const replayLabel = translate('run.rerun', { defaultValue: 'Rerun' });
```

Comment:
Seems like this should have a hyphen in it? Paging @i386

Reply:
Has anyone tested this change with a different locale? It looks if the user has it set to Spanish or German that they're going to get "Rerun" as the text. If so I am not sure the bug is really fixed via this code change. This is probably a i18n loading / timing issue but it would help to understand why... I had thought that stuff was fixed a long ways back. Maybe it will be made more resilient with the bundling changes coming.

Reply:
Sorry, I realize my comment is duplicative to some of the conversation below. I share the same concerns: `defaultValue` really shouldn't be used because it is probably just covering up a timing issue with load of i18n bundles.